File: chatwoot/typescript-node/src/CreateAnAccountExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.AccountsApi();
apiCaller.setApiKey(api.AccountsApiApiKeys.platformAppApiKey, "PLATFORM_APP_API_KEY");

const accountCreateUpdatePayload = new models.AccountCreateUpdatePayload();
accountCreateUpdatePayload.name = undefined;

apiCaller.createAnAccount(
  accountCreateUpdatePayload, // data
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling AccountsApi#createAnAccount:");
  console.log(error.body);
});

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/CreateAnAccountExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class CreateAnAccountExample
{
    fun createAnAccount()
    {
        ApiClient.apiKey["api_access_token"] = "PLATFORM_APP_API_KEY"

        val accountCreateUpdatePayload = AccountCreateUpdatePayload()

        try
        {
            val response = AccountsApi().createAnAccount(
                _data = accountCreateUpdatePayload,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling AccountsApi#createAnAccount")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling AccountsApi#createAnAccount")
            e.printStackTrace()
        }
    }
}
